src/coreclr/src/tools/aot/ILCompiler/ConfigurablePInvokePolicy.cs

```diff
@@ -52,7 +52,8 @@ public override bool GenerateDirectCall(string importModule, string methodName)
                         methodName == "LoadLibraryExW" ||
                         methodName == "GetProcAddress" ||
                         methodName == "SetLastError" ||
-                        methodName == "GetLastError")
+                        methodName == "GetLastError" ||
+                        methodName == "LocalAlloc")
```

Comment:
Add `LocalFree` here as well to make it symmetric? Also, `GetProcessHeap` and `HeapAlloc` do not need to be in the list now.